packages/orbit-components/src/List/index.tsx

```diff
@@ -27,6 +28,7 @@ const List = ({
   id,
   size = SIZES.NORMAL,
   type = TYPES.PRIMARY,
+  spacing = SPACINGS.NONE,
```

Comment:
We probably don't need the default value, if spacing is undefined, no classes will be applied.

Reply:
yes, good point. thx.